system/handlers/admin/DataManager.cfc

```diff
@@ -1035,6 +1035,53 @@ component extends="preside.system.base.AdminHandler" {
 		return renderView( view="admin/datamanager/versionNavigator", args=args );
 	}
 
+	private string function groupVersionNavigator( event, rc, prc, args={} ) {
```

Comment:
I do not think that this belongs in DataManager.cfc. It seems completely specific to system config so should go int he sysConfig admin handler I think.